### PR TITLE
Added Protectumus website security scanner

### PR DESCRIPTION
**Pull request by pascut**
Added Is it hacked ( isithacked.com ) and Protectumus ( protectumus.com )- free website security products

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -314,6 +314,8 @@ Table of Contents
   * [Shieldfy](https://shieldfy.io) — Web application firewall and vulnerability detection for developers, free plan up to 100k requests per month.
   * [Sqreen](https://www.sqreen.com/) — Application security monitoring and protection (RASP, WAF and more) for web applications and APIs. Free for 1 app and 3 million requests.
   * [Sucuri SiteCheck](https://sitecheck.sucuri.net) - Free website security check and malware scanner
+  * [Is it Hacked](http://isithacked.com) - Free cloud website security check and malware scanner. Email notifications for registered users.
```
Comment (ripienaar) [inline README.md]: this one seems super minimal and barely maintained, I dont think it crosses the bar for the kind of quality site I'd look at

File: README.md [hunk 2]
```diff
@@ -314,6 +314,8 @@ Table of Contents
...
+  * [Protectumus](https://protectumus.com) - Free website security check, site antivirus and server firewall (WAF). Email and SMS notifications for registered users.
```
Comment (ripienaar) [inline README.md]: The WAF isnt included in the free tier.

File: README.md (hunk 1 above)
Comment (pascut) [reply README.md]: Thanks for your comment, this product was designed to be minimal, but the security checks work well and we have have over 6900 registered users and over 20 000 websites in our database. So this is not a scam, this app really works well. We will add and SSL certificate to the website and we will create some more text content. This service is %100 free, which is not the case for most of the websites listed here. So please reconsider and accept it. Thanks.

File: README.md (hunk 2 above)
Comment (pascut) [reply README.md]: The WAF is included in the free tier. The free plan also provides: "Infections Scan", "Malware detection" and "Bot protection" which are part of the WAF. The paid plans offer more features, but the free plan offers basi WAF functions, which were mentioned above. Thanks for checking the website.

File: README.md (hunk 1 above)
Comment (ripienaar) [reply README.md]: I appreciate the tests might be good, however this just isnt current or maintained from outside appearances, blog posts from the 2013 etc, links to hacking news with articles from 2016 etc.

Please remove it.

Comment (pascut) [reply README.md]: I listed to you and I removed isithacked.com from the list. Thanks for your observations.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**